client/src/components/ResultsMap.js

```diff
@@ -59,21 +70,26 @@ function Map({
           stakeholders
             .filter((sh) => sh.latitude && sh.longitude)
             .map((stakeholder, index) => {
-              const isVerified = stakeholder.verifiedDate && moment.isDate(stakeholder.verifiedDate) //can we assume it will be a date of some sort?
-
+              const isVerified = !!stakeholder.verifiedDate //can we assume it will be a date of some sort?
```

Comment:
The dates should all be moment objects or null if empty
```suggestion
              const isVerified = !!stakeholder.verifiedDate //can we assume it will be a date of some sort?
```